app/js/overview.js

```diff
@@ -0,0 +1,33 @@
+document.querySelector(".overview").addEventListener("click", function() {
+    const tableBody = document.querySelector('.overview_table_body')
+    // Sets the table body content as empty in case any table body has been created before
+    tableBody.innerHTML = ''
+    let overviewData = []
+    let numOfQuestions = document.querySelectorAll('#questions .question').length
+    for (let i = 1; i <= numOfQuestions; i++) {
+        let questionObject = {}
+        questionObject['id'] = i
+        questionObject['isFlagged'] = flaggedQuestions[i]
+        questionObject['isAnswered'] = getUserAnswers()[i] == 'unanswered' ? false : true
```

Comment:
Could you replace the ternary operator with:

!= 'unanswered'

Reply:
Solved.